build.sbt

```diff
@@ -146,11 +146,16 @@ lazy val docs = project("docs")
       "akka.version" -> Dependencies.akkaVersion,
       "scala.binaryVersion" -> scalaBinaryVersion.value,
       "scala.version" -> scalaVersion.value,
+      "scaladoc.version" -> scalaVersion.value,
```

Comment:
This has to be here due to a bug in Paradox. Will try to get a fix in later today.